internal/connect/command_offset_describe.go

```diff
@@ -0,0 +1,180 @@
+package connect
+
+import (
+	"encoding/json"
+
+	connectv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect/v1"
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type serializedOffsetConnectorOut struct {
+	Id       string                                      `json:"id" yaml:"id"`
+	Name     string                                      `json:"name" yaml:"name"`
+	Offsets  []map[string]interface{}                    `json:"offsets,omitempty" yaml:"offsets,omitempty"`
+	Metadata connectv1.ConnectV1ConnectorOffsetsMetadata `json:"metadata,omitempty" yaml:"metadata,omitempty"`
+}
+
+type ConnectDescribeOut struct {
+	Id   string `human:"Id"`
+	Name string `human:"Name"`
+}
+
+type offsetsDescribeOut struct {
+	Partition string `human:"Partition"`
+	Offset    string `human:"Offset"`
+}
+
+type metadataDescribeOut struct {
+	ObservedAt string `human:"Observed At"`
+}
```

Comment:
Any reason we aren't supporting `serialized` output in addition to `human` output?